### DISCUSSION TITLE
use of vtkLODActor in place of vtkActor to enable larger meshes

### DISCUSSION THREAD

**Discussion by AndrewAnnex** [pyvista/pyvista#3128] [Ideas]
I was curious about if vtk had a built in way to do LOD loading and found that it had (as paraview does this). It looks like the vtkLODActor is able to be used in place of vtkActor for this, and it looks like there are a number of ways to control the decimation of meshes and etc. I haven't had time to make a POC but it appears that there is another python vtk library that is using it. Could be worth looking into 

https://vtk.org/doc/nightly/html/classvtkLODActor.html
https://fury.gl/v0.1.x/_modules/fury/actor.html

**Comment by akaszynski** [comment]
That's really cool! That's something we can add at a later date, and thanks for pointing out the docs and example source!

**Comment by ConnorNoddin** [comment]
This feature would be highly beneficial to my Pyvista project

**Comment by user27182** [comment]
This feature request is also tracked in https://github.com/pyvista/pyvista/issues/3471
